frameit/frames_generator/Rakefile

```diff
@@ -79,4 +100,36 @@ def convert_image_resource(path)
   image.write(path)
 end
 
+def measure_slot(path)
+  tmp_output = Time.now.to_i.to_s + ".png"
+
+  convert = MiniMagick::Tool::Convert.new
+  convert << path
+  convert.negate
+  convert.define("connected-components:verbose=true")
+  convert.define("connected-components:area-threshold=100")
+  convert.connected_components("8")
+  convert.auto_level
+  convert << tmp_output
+  command_output = convert.call
+
+  sh("rm #{tmp_output}", verbose: false)
+
+  raw_dimensions = command_output.split("\n")[1].strip
+  dimensions = /(\d+)x(\d+)\+(\d+)\+(\d+)/.match(raw_dimensions)
```

Comment:
I assumed that `x` and `y` offsets are always non-negative. Do you think we should support both: non-negative and negative?

Reply:
Not until this explodes - then we can handle it.